orchestrator/cli/migrate_workflows.py

```diff
@@ -0,0 +1,230 @@
+    keys = keys or _enumerate_menu_keys(options_list)
+    done = False
+    while not done:
+        for k, txt_v in zip(keys, options_list):
+            _noqa_print(f"{k}) {txt_v[0]}")
+        choice = get_user_input("? ")
+        if choice not in keys:
+            _noqa_print("Invalid choice")
+            done = not repeat
+        else:
+            return options_list[keys.index(choice)][1]
+
+
+def _print_workflows_table(workflows: List[WorkflowTable]):
+    items = [(wf.name, wf.target, wf.description, wf.products[0].product_type) for wf in workflows if wf.products]
+    print_fmt(
+        tabulate(
+            items,
+            headers=["#", "name", "target", "description", "product_type"],
+            showindex=itertools.count(1),
+        ),
+        end="\n\n",
+    )
+
+
+def _add_workflow(workflows: Dict[str, LazyWorkflowInstance], state: dict) -> dict:
+    print_fmt("\nAdd new workflow\n", flags=[COLOR.UNDERLINE])
+
+    if not workflows:
+        _noqa_print("No registered workflows found to add to the database")
+        return state
+
+    registered_product_types = [
+        row[0] for row in ProductTable.query.with_entities(ProductTable.product_type).distinct()
+    ]
+    if not registered_product_types:
+        _noqa_print("No registered product types found")
+        return state
+
+    _noqa_print("Which product type should the workflow be added to?")
+    product_type = _prompt_user_menu(
+        [*[(p, p) for p in registered_product_types], ("cancel", None)],
+        keys=[*_enumerate_menu_keys(registered_product_types), "q"],
+    )
+
+    if not product_type:
+        # Menu cancelled
+        return state
+
+    _noqa_print(f"\nAdd product type {str_fmt(product_type, flags=[COLOR.BOLD])} to which workflow?")
+
+    already_used_workflows = {wf["name"] for wf in state["workflows_to_add"] + state["workflows_to_delete"]}
+    wf_options = [(wf, wf) for wf in workflows if wf not in already_used_workflows]
+    wf_name = _prompt_user_menu([*wf_options, ("cancel", None)], keys=[*_enumerate_menu_keys(wf_options), "q"])
+    if not wf_name:
+        # Menu cancelled
+        return state
+
+    wf_inst = get_workflow(wf_name)
+    wf_target = wf_inst.target.value
+    wf_description = wf_inst.description
+    wf_to_add = {"name": wf_name, "target": wf_target, "description": wf_description, "product_type": product_type}
+    return {**state, "workflows_to_add": [*state["workflows_to_add"], wf_to_add]}
+
+
+def _delete_workflow(workflows: List[WorkflowTable], state: dict) -> dict:
+    print_fmt("\nDelete existing workflow\n", flags=[COLOR.UNDERLINE])
+    already_used_workflows = {wf["name"] for wf in state["workflows_to_add"] + state["workflows_to_delete"]}
+    items = [
+        (wf.name, wf.target, wf.description, wf.products[0].product_type)
+        for wf in workflows
+        if wf.products and wf.name not in already_used_workflows
+    ]
+    keys = ["#", "name", "target", "description", "product_type"]
```

Comment:
unsorted gets messy with a lot of workflows, sort the workflows by product type and target.

```suggestion
    ]
    items = sorted(items, key=operator.itemgetter(3, 1))
    keys = ["#", "name", "target", "description", "product_type"]
```